src/test/groovy/com/commercehub/gradle/plugin/avro/AvroPluginFunctionalSpec.groovy

```diff
@@ -118,14 +118,14 @@ class AvroPluginFunctionalSpec extends FunctionalSpec {
     def "gives a meaningful error message when presented a malformed schema file"() {
         given:
         copyResource("enumMalformed.avsc", avroDir)
-
+        def errorFilePath = new File("src/main/avro/enumMalformed.avsc").path
```

Comment:
I'm going to assume that this (and similar changes) were made because the tests failed on Windows, and these were the changes needed to make them work on that platform.  Is that correct?  If so, my apologies; I know better, but overlooked it.

Reply:
This is correct. Done for Windows.

Reply:
Awesome.  Thank for you for confirming.  Given that context, the changes made look appropriate.